praw/praw.ini

```diff
@@ -68,3 +68,4 @@ message_kind: t7
 permalink_domain: reddit.local
 submission_kind: t6
 subreddit_kind: t5
+
```

Comment:
Not sure why this line was added, nevertheless, `validate_certs` should be set in the default site with the value True.